### PR TITLE
Project 2 Ann-Sofi

### PR DESCRIPTION
**Pull request by Ann-Sofi**
I had a hard time with flexbox to understand how it works and how things impact each other. It becomes more difficult when there is nesting involved and when the child becomes a parent.

I have a diffrent structure for my big news card from the small  news card because in the big news card I had to make one div for img and one for text to make them placed in a row in desktop.
I had the same structure for the small news card but when I inspected they were not one card together and I had trouble with things like overlapping. When I asked for help I got the tip to remove one div and have both text and img in the same. Then it worked fine to place them. But Im not sure if I have done it correctly? Is there a smarter way that I am missing?

### CODE REVIEW

**Comment by Ann-Sofi** [comment]
Instead of a hamburger menu for mobile i made mine a pancake-menu  ;-) That is why it is displayed in the mobile version and not in desktop. If I have had time a would have made it a real menu because at this pont it is not very clear why you can click on it.

File: code/index.html [hunk 1]
```diff
@@ -1,22 +1,94 @@
...
+      <link rel="preconnect" href="https://fonts.gstatic.com">
+      <link href="https://fonts.googleapis.com/css2?family=Open+Sans&display=swap" rel="stylesheet">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    </head>
+    <body>
+
+      <!--HEADER: LOGO & NAVBAR-->
+      <header class="header container">
+        <a href="#">
+        <img class="logo" src="./img/iconpancake.JPG" alt="pancake icon"></a>
```
Comment (EmOlsz) [inline code/index.html:16]: Wrong indentations - let's fix those line breaks!

File: code/style.css [hunk 2]
```diff
@@ -1,4 +1,211 @@
...
+  margin: 0;
+}
+
+/****HEADER****/
+.header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.headline p, h1 {
```
Comment (EmOlsz) [inline code/style.css:17]: I'm not sure if that's what you wanted to achieve - if you would like to apply `text-align: center` value to both `p` and `h1` nested inside `.headline`, then you need to fix you reference to this one :
```
.headline p, .headline h1 {
  text-align: center;
}
```

**Review by EmOlsz — CHANGES_REQUESTED** [review]
Hey Ann!
Pancake menu sounds far better than hamburger one, I'm in your team already ;)

Your project looks pretty good, I don't have many things to point out. Nonetheless, let's fix those indentations before approval.

We have prepared for all of you a document with complete set of feedback regarding ideal project for this week, check it out and try to take those tips with you to next projects!
https://www.notion.so/technigo/General-feedback-for-class-Jan-2021-ac7821ff3e1849c3bcac26f5d9888230

**Review by EmOlsz — APPROVED** [review]
Thank you for providing changes, project approved :)

**Pull request closed without merging**